automation-roles/30-provision-infra/provision-aws/tasks/provision-ipi.yml

```diff
@@ -28,6 +28,14 @@
       AWS_ACCESS_KEY_ID: "{{ _aws_access_key }}"
       AWS_SECRET_ACCESS_KEY: "{{ _aws_secret_access_key }}"
 
+  - name: Store the metadata json file as secret {{ environment_name }}-metadata-json in group {{ environment_name }}
+    include_role:
+      name: vault-set-secret
+    vars:
+      secret_name: "{{ environment_name }}-metadata-json"
```

Comment:
The secret name must be linked to the OpenShift cluster name, not the environment name.